src/depends/NAT/nat.cpp

```diff
@@ -0,0 +1,169 @@
+#include "nat.h"
```

Comment:
is this our code? if no, we may have to credit or check the licence
if yes, add our copyright comments